modules/ossm-deploying-jaeger.adoc

```diff
@@ -0,0 +1,104 @@
+====
+
+* *production* - The production strategy is intended for production environments, where long term storage of trace data is important, as well as a more scalable and highly available architecture is required. Each of the backend components is therefore deployed separately.  The Agent can be injected as a sidecar on the instrumented application. The Query and Collector services are configured with a supported storage type - currently Elasticsearch. Multiple instances of each of these components can be provisioned as required for performance and resilience purposes.
+
+* *streaming* - The streaming strategy is designed to augment the production strategy by providing a streaming capability that effectively sits between the Collector and the backend storage (Elasticsearch). This provides the benefit of reducing the pressure on the backend storage, under high load situations, and enables other trace post-processing capabilities to tap into the real time span data directly from the streaming platform (https://access.redhat.com/documentation/en-us/red_hat_amq/7.6/html/using_amq_streams_on_openshift/index[AMQ Streams]/ https://kafka.apache.org/documentation/[Kafka]).
+
+[NOTE]
+====
+The streaming strategy requires an additional Red Hat subscription for AMQ Streams.
+====
+
+[id="ossm-deploying-jaeger-default_{context}"]
+== Default Jaeger deployment
+
+To use the default `allInOne` Jaeger deployment strategy set `spec:addons:jaeger:install:storage:type` to `Memory`. You can accept the defaults or specify additional configuration options under `install`.  If you do not specify Jaeger configuration options, the Control Plane will use the `allInOne` deployment strategy by default.
+
+.Control Plane default Jaeger parameters (Memory)
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+spec:
+  version: v2.0
+  tracing:
+    sampling: 10000
+    type: Jaeger
+  addons:
+    jaeger:
+      name: jaeger
+      install:
+        storage:
+          type: Memory
+----
+
+[id="ossm-deploying-jaeger-production_{context}"]
+== Production Jaeger deployment
+
+To use the `production` deployment strategy, set  `spec:addons:jaeger:install:storage:type` to 'Elasticsearch' and specify additional configuration options under `install`.   Or you can create and configure your Jaeger instance and set  `spec:addons:jaeger:name` to the name of the Jaeger instance, for example  `jaeger-production`.
+
+.Control Plane default Jaeger parameters (Elasticsearch)
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: production
+spec:
+  version: v2.0
+  tracing:
+    sampling: 10000
```

Comment:
This is not the default sampling rate.  The default value is actually 100 (1%).

Reply:
The value `sampling;10000` is what I'm seeing in the OpenShift console when creating a v2 SMCP.  Is that a bug?

Reply:
ahh.  Well, not a bug, but not _the_ default value, but _a_ default value.  The resource in the console may be initialized to that, but if it is unspecified, e.g. if the user deletes that field, the value used is 100.  I don't think this is an issue with the documentation, just may be confusing when the heading for the example includes the word, "default."  I'm okay with the changes, as they are, hence my approving this PR.